core/db/curibio/alembic/versions/2ccc9fe21ec1_adding_plan_end_date_to_usage_.py

```diff
@@ -0,0 +1,30 @@
+"""Adding plan end date to usage_restrictions json object structure
+
+Revision ID: 2ccc9fe21ec1
+Revises: 1f23e047d2b0
+Create Date: 2023-01-25 14:40:08.612570
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "2ccc9fe21ec1"
+down_revision = "1f23e047d2b0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    new_usage_restrictions_struct = {"pulse3d": {"jobs": -1, "uploads": -1, "end": None}}
```

Comment:
This should just be written directly into query instead of created as a python dict and inserted into an f-string

Reply:
Also we should use a more descriptive name than "end", something like "expiration_date" would be better